lib/ret/media_search.ex

```diff
@@ -20,47 +20,249 @@ defmodule Ret.MediaSearch do
   alias Ret.{Repo, OwnedFile, Scene, SceneListing}
 
   @page_size 24
+  @max_face_count 60000
+
+  def search(%Ret.MediaSearchQuery{source: "scene_listings", cursor: cursor, filter: "featured", q: query}) do
+    scene_listing_search(cursor, query, "featured", asc: :order)
+  end
 
-  def search(%Ret.MediaSearchQuery{source: "scene_listings", page: page, filter: "featured", q: query}) do
-    scene_listing_search(page, query, "featured", asc: :order)
+  def search(%Ret.MediaSearchQuery{source: "scene_listings", cursor: cursor, filter: filter, q: query}) do
+    scene_listing_search(cursor, query, filter)
   end
 
-  def search(%Ret.MediaSearchQuery{source: "scene_listings", page: page, filter: filter, q: query}) do
-    scene_listing_search(page, query, filter)
+  def search(%Ret.MediaSearchQuery{source: "sketchfab", cursor: cursor, filter: "featured", q: q}) do
+    query =
+      URI.encode_query(
+        type: :models,
+        downloadable: true,
+        count: @page_size,
+        face_count: @max_face_count,
+        processing_status: :succeeded,
+        cursor: cursor,
+        collection: "ec06ae45eba24bfdb1278b223f8e289c",
+        q: q
```

Comment:
license options here seem tricky because there are a bunch of filters we may want to include. i think i'll defer to revealing the license in the (pending) info popup i'm planning on building into the media search